pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -233,7 +326,11 @@ func (db *DB) GetBandwidthAllocations() (map[storj.NodeID][]*Agreement, error) {
 		rbaBytes := []byte{}
 		agreement := &Agreement{}
 		var satellite []byte
-		err := rows.Scan(&satellite, &rbaBytes, &agreement.Signature)
+		var uplink []byte
+		var serialnum string
+		var total, maxsize, createdUnixSec, expirationUnixSec, daystartdateUnixSec int64
+		var action string
+		err := rows.Scan(&satellite, &rbaBytes, &agreement.Signature, &uplink, &serialnum, &total, &maxsize, &createdUnixSec, &expirationUnixSec, &action, &daystartdateUnixSec)
```

Comment:
You don't have to scan all columns. You can limit number of columns in ```rows, err := db.DB.Query(`SELECT * FROM bandwidth_agreements ORDER BY satellite`)``` above.

Reply:
done